packages/db-mongodb/src/transactions/rollbackTransaction.ts

```diff
@@ -17,7 +17,11 @@ export const rollbackTransaction: RollbackTransaction = async function rollbackT
   }
 
   // the first call for rollback should be aborted and deleted causing any other operations with the same transaction to fail
-  await this.sessions[id].abortTransaction()
-  await this.sessions[id].endSession()
+  try {
+    await this.sessions[id].abortTransaction()
+    await this.sessions[id].endSession()
+  } catch (error) {
+    // ignore the error as it is likely a race condition from multiple errors
```

Comment:
`await this.sessions[id].endSession()` may need to be in the catch block.

Reply:
You might be right about, if the abortTransaction errors endSession won't be called.